capa/v31.0.0/README.md

```diff
@@ -0,0 +1,225 @@
+- Cilium v1.17.4 disables kubernetes api connectivity check for liveness probes. (Upstream PR: https://github.com/cilium/cilium/pull/38703)
+- Upgrade Cilium to [v1.17.3](https://github.com/cilium/cilium/releases/tag/v1.17.3).
+- Upgrade Cilium to [v1.17.2](https://github.com/cilium/cilium/releases/tag/v1.17.2).
+- Remove cleanup kube-proxy patch.
+- Identity computation label exclusion list regular expressions. Remove `controller-uid`, since this is excluded by default now.
+- Upgrade Cilium to [v1.17.0](https://github.com/cilium/cilium/releases/tag/v1.17.0).
+- Use upstream default value for `prometheus.metrics`.
+- Enable Envoy Proxy in standalone DaemonSet.
+
+### cilium-crossplane-resources [v0.2.0...v0.2.1](https://github.com/giantswarm/cilium-crossplane-resources/compare/v0.2.0...v0.2.1)
+
+#### Added
+
+- Included the `giantswarm.io/cluster` label
+
+### cloud-provider-aws [v1.30.8-gs1...v1.31.5-gs1](https://github.com/giantswarm/aws-cloud-controller-manager-app/compare/v1.30.8-gs1...v1.31.5-gs1)
+
+#### Changed
+
+- Chart: Update to upstream v1.31.5.
+
+### cluster-autoscaler [v1.30.4-gs1...v1.31.2-gs2](https://github.com/giantswarm/cluster-autoscaler-app/compare/v1.30.4-gs1...v1.31.2-gs2)
+
+#### Added
+
+- Add additional labels to ignore during ASG balancing check
+- Support adding additional labels to the `PodMonitor` resource via the `podMonitor.additionalLabels` value.
+
+#### Changed
+
+- Chart: Update to upstream v1.31.2.
+
+### coredns [v1.24.0...v1.25.0](https://github.com/giantswarm/coredns-app/compare/v1.24.0...v1.25.0)
+
+#### Changed
+
+- Update `coredns` image to [1.12.1](https://github.com/coredns/coredns/releases/tag/v1.12.1).
+
+### etcd-defrag [v1.0.2...v1.0.4](https://github.com/giantswarm/etcd-defrag-app/compare/v1.0.2...v1.0.4)
+
+#### Changed
+
+- Chart: Update dependency ahrtr/etcd-defrag to v0.27.0. ([#29](https://github.com/giantswarm/etcd-defrag-app/pull/29))
+- Chart: Update dependency ahrtr/etcd-defrag to v0.26.0. ([#22](https://github.com/giantswarm/etcd-defrag-app/pull/22))
+
+### etcd-k8s-res-count-exporter [v1.10.3...v1.10.4](https://github.com/giantswarm/etcd-kubernetes-resources-count-exporter/compare/v1.10.3...v1.10.4)
+
+#### Fixed
+
+- Fix linting issues.
+- Go: Update dependencies.
+
+### k8s-audit-metrics [v0.10.2...v0.10.3](https://github.com/giantswarm/k8s-audit-metrics/compare/v0.10.2...v0.10.3)
+
+#### Fixed
+
+- Fix linting issues.
+- Go: Update dependencies.
+
+### net-exporter [v1.22.0...v1.23.0](https://github.com/giantswarm/net-exporter/compare/v1.22.0...v1.23.0)
+
+#### Changed
+
+- Check for errors when closing connections.
+- Switch from Endpoints to EndpointSlices for neighbors discovery.
+
+### node-exporter [v1.20.2...v1.20.3](https://github.com/giantswarm/node-exporter-app/compare/v1.20.2...v1.20.3)
+
+#### Changed
+
+- Go: Update dependencies.
+
+### observability-bundle [v1.11.0...v2.0.0](https://github.com/giantswarm/observability-bundle/compare/v1.11.0...v2.0.0)
```

Comment:
@giantswarm/team-atlas Can you please confirm this major bump? I saw v2.0.0 got released yesterday and do not know if it's intended to be in v31.0.0 already or if we need customers to go for the latest v1.x.x (at the time of writing this: v1.16.1) before upgrading to v2.0.0. We had some severe issues with breaking changes in `security-bundle` recently and I'd just like to make sure the upgrade path works in production.